lucien-ios/Lending/LendingTableViewCell.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  LendingTableViewCell.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/20/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+class LendingTableViewCell: UITableViewCell {
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+
+    override func prepareForReuse() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Overridden methods call super Violation: Method 'prepareForReuse()' should call to super function [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Aoverridden_super_call)